src/gtfs/read.rs

```diff
@@ -584,6 +604,34 @@ fn manage_comment_from_stop(
     comment_links
 }
 
+fn manage_odt_comment_from_stop_time(
+    comments: &mut CollectionWithId<objects::Comment>,
+    companies: &mut CollectionWithId<objects::Company>,
+    on_demand_transport_comment: Option<String>,
```

Comment:
This whole function is useless to call if this parameter is `None`. It feels that this function should be conditionally called. Therefore, this parameter should be `String`, not `Option<String>`. Same comment could be made for companies. I'm wondering if this function should be more like this.

```rust
fn manage_odt_comment_from_stop_time(
    comments: &mut CollectionWithId<objects::Comment>,
    on_demand_transport_comment: String,
    company: &Company,
) -> Option<String> {
```